docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,390 +1,565 @@
+| `PremultipliedAlpha`| `bool`                      | `false`                          | Enables (`true`) or disables (`false`) the premultiplied alpha.                                                   |
+| `VisualFittingMode` | `VisualFittingModeType`     | `FitKeepAspectRatio` - for AnimatedImageVisual, MeshVisual, PrimitiveVisual, TextVisual<br>`Fill` - otherwise | Specifies the visual fitting mode (possible values: `FitKeepAspectRatio`, `Fill`, `OverFitKeepAspectRatio`, `Center`, `FitHeight`, `FitWidth`). |
+| `CornerRadius`      | `float`                     | `0.0f`                           | The visual corner radius (rounded for values > 0).                                                                |
 
-    The following example illustrats part of the [Tizen.NUI.ColorVisual](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.ColorVisual.html) class, which defines a color visual property map:
 
-    ```
-    public class ColorVisual : VisualMap
-
-    private Color _mixColorForColorVisual = null;
-
-    public Color Color
-    {
-        get
-        {
-            return _mixColorForColorVisual;
-        }
-        set
-        {
-            _mixColorForColorVisual = value;
-            UpdateVisual();
-        }
-    }
-    ```
+<!---------------------------------------------------------------------------------------------------------------------------->
+<a name="visualTypes"></a>
+## Visual types provided by NUI
 
--   You can create a visual from an output visual map:
+The following visual types are available in NUI:
+[Border](#borderVisual),
+[Color](#colorVisual),
+[Gradient](#gradientVisual),
+[Image](#imageVisual),
+[NPatch](#npatchVisual),
+[SVG](#svgVisual),
+[AnimatedImage](#animatedImageVisual),
+[Mesh](#meshVisual),
+[Primitive](#primitiveVisual) and
+[Text](#textVisual)
 
-    ```
-    var colorMap = new ColorVisual{Color=Color.White;};
-    var _colorVisual = VisualFactory.Instance.CreateVisual(colorMap.OutputVisualMap);
-    RegisterVisual(ColorVisualPropertyIndex, _colorVisual);
-    ```
+<!---------------------------------------------------------------------------------------------------------------------------->
+<a name="borderVisual"></a>
+### BorderVisual
 
-    You can also create a visual using an output map in a control property. The following example uses the `Background` property:
+The rectangular frame with a given thickness. The whole frame is plotted inside the area designated by the `BorderVisual` size as long as it is possible, e.g.
+- the `BorderVisual` of size 200x200 and the border width of 100, appears as a square of the size of 200x200, having the color of the border,
+- the `BorderVisual` of size 100x200 and the border width of 200, appears as a rectangle of the size of 300x200, having the color of the border.
```

Comment:
Can you please verify this update? 
I think it is incorrectly updated.

Reply:
It's correct, but since you're asking, it probably means I should add a picture, right?